docs/build-acceleration.md

```diff
@@ -49,6 +49,8 @@ You may disable build acceleration for specific projects in your solution by red
 
 ## Debugging
 
+### Enable logging
+
 Build acceleration runs with the FUTDC, and outputs details of its operation in the build log. To enable this logging:
 
 > Tools | Options | Projects and Solutions | .NET Core
```

Comment:
I think you mean:

```suggestion
> Tools | Options | Projects and Solutions | SDK-Style Projects
```

Reply:
Also in the caption of the image below this line, but I couldn't leave a comment on that. 🤷🏻‍♂️

Reply:
Good catch. This was copied from the FUTDC document, so I'll update it there too.